README.md

```diff
@@ -118,4 +120,5 @@ I would like to thank Microverse for the learning materials provided for this ex
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 📝 License
-This project is MIT licensed.
+This project is [MIT](./LICENSE) licensed.
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] You did a good job following the previous reviewer's advice. 💯 However, your link is broken because you don't have any LICENSE files yet. Please consider following this [tutorial](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) in order to add one (You could use MIT) and make sure the link in your README is working before submitting your project again! 🤓 🛠️

Reply:
Thanks for the tutorial reference. I would fix this